### ISSUE TITLE
[gRPC] Loader option causes ClientGrpc.getService fail

### ISSUE THREAD

**Issue by mxth** [nestjs/nest#1013]
https://github.com/nestjs/nest/blob/d0bd2f7b200ebae72b29bc6b5fc3d62db81cd8fc/packages/microservices/client/client-grpc.ts#L36

> (node:1600) UnhandledPromiseRejectionWarning: TypeError: Channel options must be an object with string keys and integer or string values

The message is pretty clear on what causes it. Right now my workaround is removing `loader` attribute before passing it to the constructor.

**Comment by kamilmysliwiec** [comment]
Could you share your entire `loader` options?

**Comment by mxth** [comment]
@kamilmysliwiec Sure, here it is

`loader: {
        includeDirs: [join(__dirname, process.env.PROTOS_PATH || '')]
      }`

**Comment by kamilmysliwiec** [comment]
Honestly, just tried your code and I cannot reproduce your issue. Do you use the latest version of the framework?

**Comment by nooop3** [comment]
i have the same issue when set loader, solutions:
1. use absolute path for protoPath;
2. don't pass loader property in nestjs client-grpc getService method.

**Comment by mxth** [comment]
@kamilmysliwiec I did at the time. It was `5.2.2`. I haven't tried again with the new version yet.

@nooop3 I need it because of the imports in protobuf files.

**Comment by mxth** [comment]
Problem still persists with version `5.3.6`

**Comment by dzzzzzy** [comment]
```
@Client({
        transport: Transport.GRPC,
        options: {
            url: 'localhost:50051',
            package: 'notadd_module_user',
            protoPath: join(__dirname, './protobufs/user-module.proto'),
            loader: {
                arrays: true
            }
        }
    })
    public readonly userModuleClient: ClientGrpc;
```

same error:

```
Channel options must be an object with string keys and integer or string values
```

**Comment by kamilmysliwiec** [comment]
Hi guys,
I really need the whole repo with dir structure. I can't reproduce this issue without it

**Comment by dzzzzzy** [comment]
> Hi guys,
> I really need the whole repo with dir structure. I can't reproduce this issue without it

@kamilmysliwiec Here is the reproduce repo with nest sample-04-grpc `https://github.com/dzzzzzy/nest/tree/master/sample/04-grpc`

I do these changes:

1. update the `core`, `common` and `microservice` package to latest version
2. add loader options to `grpc-client.options.ts` such as:
```
loader: {
      arrays: true,
      keepCase: true,
      longs: String,
      enums: String,
      defaults: true,
      oneofs: true
    }
```
And then run `npm start `, this issues can reproduced.

**Comment by kamilmysliwiec** [comment]
Thanks @dzzzzzy. Looks like only the Client throws an exception when `loader` option is passed. Just published 5.3.7 fix :)
